common/k8stest/util_rdma.go

```diff
@@ -3,11 +3,16 @@ package k8stest
 import (
 	"encoding/json"
 	"fmt"
+	"strings"
 
+	"github.com/openebs/openebs-e2e/common/e2e_agent"
 	agent "github.com/openebs/openebs-e2e/common/e2e_agent"
+	"github.com/openebs/openebs-e2e/common/e2e_config"
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+const RdmaDeviceName = "rxe0"
```

Comment:
Should this be hard-coded?

Reply:
Does convention dictate that it should be rxeN?

Reply:
> Should this be hard-coded?

yes, it can be hardcoded or dynamically generated

Reply:
> Does convention dictate that it should be rxeN?

no, it can be anything